src/main/resources/application.properties

```diff
@@ -16,7 +16,7 @@ azure.keyvault.enabled=false
 azure.keyvault.uri=https://default.vault.localhost
 wrongsecret-3=if_you_see_this_please_use_Azure_Setup
 secretmountpath=/mnt/secrets-store
-challengedockermtpath=/var/tmp/helpers
+challengedockermtpath=/Users/mno/workspace/opensource/wrongsecrets
```

Comment:
i believe this is used for something else already, can you use a new variable please :)?

Reply:
Reverted